newthread.php

```diff
@@ -1037,7 +1037,7 @@
 				$post_captcha->build_hcaptcha();
 			}
 		}
-		else if($correct && (in_array($post_captcha->type, array(4, 5))))
+		else if($correct && (in_array($post_captcha->type, array(4, 5, 8))))
```

Comment:
Not exactly related to this PR, but I think we should introduce some constants/static variables to define these captcha types rather than having these types of checks scattered throughout the codebase.

Reply:
That makes much more sense and I will include in the issue I open to track the other code improvement.